src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/PurposesApiServiceImpl.scala

```diff
@@ -79,6 +79,31 @@ final case class PurposesApiServiceImpl(
     }
   }
 
+  override def createPurposeVersion(purposeId: String, purposeVersionSeed: PurposeVersionSeed)(implicit
+    contexts: Seq[(String, String)],
+    toEntityMarshallerProblem: ToEntityMarshaller[Problem],
+    toEntityMarshallerCompactPurposeVersion: ToEntityMarshaller[CompactPurposeVersion]
+  ): Route = {
+    logger.info(s"Create purpose version $purposeId with seed $purposeVersionSeed")
```

Comment:
we generally do not blindly log the seed to avoid situations where future changes add sensitive data that would be printed in the logs.
If you prefer, you can still log the daily calls, but let's do it explicitly
```suggestion
    logger.info(s"Create purpose version $purposeId with dailyCalls ${purposeVersionSeed.dailyCalls}")
```

Reply:
just to have a less ambiguous log
```suggestion
    logger.info(s"Creating version for purpose $purposeId with seed $purposeVersionSeed")
```